test.py

```diff
@@ -19,3 +19,6 @@
 try: shutil.rmtree("../restricted-functions")
 except: pass
 else: raise Exception("failed")
+try: open("w", "setup.py").write("text")
+except: pass
+else: raise Exception("failed")
```

Comment:
@donno2048 maybe this should be replaced with a call to `print` to the rest of the program can be tested
```suggestion
else: print("Failed to block wirte")
```